research-hub-web/src/app/components/articles/articles.component.html

```diff
@@ -1,116 +1,201 @@
-                            article.relatedOrgsCollection.items.length > 0"
-                        fxLayout="column" fxFlex="20%" fxFlexOrder="1" fxFlexOrder.lt-md="2">
-
-                        <!-- Contacts -->
-                        <ng-container *ngTemplateOutlet="relatedContacts; context: { flex: 'column', hide: 'true' }"></ng-container>
-
-                        <!-- Official Documents -->
-                        <ng-container *ngTemplateOutlet="officialDocuments; context: { flex: 'column', hide: 'true'}"></ng-container>
-
-                        <!-- Organizations -->
-                        <ng-container *ngTemplateOutlet="relatedOrganisations; context: { flex: 'column', hide: 'true'}"></ng-container>
-
-                    </div>
-
-                    <!-- Related items -->
-                    <div *ngIf="article.relatedItemsCollection.items.length > 0" fxLayout="column" fxFlex="20%" fxFlexOrder="3">
-                        <ng-container *ngTemplateOutlet="relatedItems; context: { flex: 'column', hide: 'true'}"></ng-container></div>
-                </div>
-            </div>
+        <h3 *ngIf="article?.relatedDocsCollection.items.length > 0">
+          Documents
+          <hr />
+        </h3>
+        <div
+          id="documents"
+          fxLayout.lg="column"
+          fxLayout.lt-lg="row wrap"
+          fxLayoutGap.lt-lg="25px"
+          fxLayout.lt-sm="column"
+          fxLayoutGap.lt-sm="0px"
+        >
+          <app-document-card
+            *ngFor="let document of article.relatedDocsCollection.items"
+            [document]="document"
+          ></app-document-card>
+        </div>
+        <h3 *ngIf="article?.relatedOrgsCollection.items.length > 0">
+          Organisations
+          <hr />
+        </h3>
+        <div
+          fxLayout.lg="column"
+          fxLayout.lt-lg="row wrap"
+          fxLayoutGap.lt-lg="25px"
+          fxLayout.lt-sm="column"
+          fxLayoutGap.lt-sm="0px"
+        >
+          <app-org-unit-card
+            *ngFor="let orgUnit of article.relatedOrgsCollection.items"
+            [orgUnit]="orgUnit"
+          ></app-org-unit-card>
+        </div>
+      </div>
+
+      <!-- Related items -->
+      <div
+        *ngIf="article.relatedItemsCollection.items.length > 0"
+        fxFlex.gt-lg="0 0 350px"
+        fxLayout="column"
+        fxFlexOrder="3"
+      >
+        <h3>
+          Explore Related
+          <hr />
+        </h3>
+        <div
+          id="you-might-be-interested-in"
+          fxLayout.lg="column"
+          fxLayout.lt-lg="row wrap"
+          fxLayoutGap.lt-lg="25px"
+          fxLayout.lt-sm="column"
+          fxLayoutGap.lt-sm="0px"
+        >
+          <app-standard-card
+            *ngFor="let content of article.relatedItemsCollection.items"
+            [contentItem]="content"
+          ></app-standard-card>
         </div>
-    <div class="section"></div>
```

Comment:
The section class adds some top and bottom margin - looks like it's missing from articles (and maybe other page types?) now - see the bottom of the page on mobile view. Just need to add that margin back in somewhere.

Reply:
Ahh that's what this was for. I'm going to add the ```margin-bottom``` class to the top level div in each content page